src/Build.UnitTests/Evaluation/Expander_Tests.cs

```diff
@@ -2800,34 +2800,23 @@ public void PropertyFunctionVersionComparisonsFailsWithInvalidArguments(string b
             var expander = new Expander<ProjectPropertyInstance, ProjectItemInstance>(pg, FileSystems.Default);
             string expectedMessage = ResourceUtilities.GetResourceString("InvalidVersionFormat");
 
-            AssertThrows($"$([MSBuild]::VersionGreaterThan('{badVersion}', '1.0.0'))");
-            AssertThrows($"$([MSBuild]::VersionGreaterThan('1.0.0', '{badVersion}'))");
+            AssertThrows(expander, $"$([MSBuild]::VersionGreaterThan('{badVersion}', '1.0.0'))", expectedMessage);
```

Comment:
Why did you have to change all these? It'll make `git blame` harder.

Reply:
Reusing code. I had to change the function signature to make it available for more than one test.

Reply:
I mean, I do think it's good to reuse code, but the three instances I see where you used your new method, you started the method with the exact same two lines. Why not put those in the same method? Note that this is a test, so it doesn't have to be optimized. (And having theories the way you (and nguerrera) did effectively does the same thing.)

Reply:
I don't have a concern with this change. It might be a good idea to make a class-level `expander` for use in multiple tests if they all need the same one (like for all the ones that don't need properties and don't touch the filesystem). But that can wait for a test-perf effort.